base/float.jl

```diff
@@ -541,7 +541,7 @@ function float{T}(A::AbstractArray{T})
     convert(AbstractArray{typeof(float(zero(T)))}, A)
 end
 
-for fn in (:float,:big)
+for fn in (:float,)
```

Comment:
Absolutely. I anticipate removing this block altogether when rebasing with #18495 in (or removing the corresponding block from #18495 when rebasing #18495 with this PR in, whichever occurs).